src/providers/php/mod.rs

```diff
@@ -61,17 +61,31 @@ impl PhpProvider {
             Pkg::new("nginx"),
             Pkg::new(&format!("{}Packages.composer", &php_pkg)),
         ];
-        if let Ok(php_extensions) = PhpProvider::get_php_extensions(app) {
-            for extension in php_extensions {
-                pkgs.push(Pkg::new(&format!("{}Extensions.{extension}", &php_pkg)));
-            }
-        }
+        let ext_pkgs = if let Ok(php_extensions) = PhpProvider::get_php_extensions(app) {
+            php_extensions
+                .iter()
+                .map(|extension| format!("{}Extensions.{extension}", &php_pkg))
+                .collect()
+        } else {
+            vec![]
+        };
 
         if app.includes_file("package.json") {
             pkgs.append(&mut NodeProvider::get_nix_packages(app, env)?);
         }
 
-        Ok(Phase::setup(Some(pkgs)))
+        {
+            let mut tmp_ext_pkgs = ext_pkgs.iter().map(|pkg| Pkg::new(pkg)).collect();
+            pkgs.append(&mut tmp_ext_pkgs);
+        }
+
+        let mut phase = Phase::setup(Some(pkgs));
+
+        phase.add_pkgs_libs(ext_pkgs);
+
+        phase.add_apt_pkgs(vec!["default-mysql-client".into()]);
```

Comment:
Should this always be added? Can we detect if mysql is being used?

Reply:
Since Laravel doesn't seem to provide a good way to detect what database is being actually used without severe regex abuse (which would quite likely break regularly), the way to do it would be to detect environment variables. I'm nervous about doing that at build time, though, since people might be building and running images in different environments.

Reply:
Hmm, are you sure there isn't a specific file we can check? Or just a regex we can use to search everyfile?